src/assisted_test_infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -40,6 +40,9 @@ def cluster_name(self) -> str:
         warnings.warn("cluster_name is deprecated. Use Controller.entity_name instead.", DeprecationWarning)
         return self._entity_name.get()
 
+    def _get_params_from_config(self):
```

Comment:
add type annotation

Reply:
OK, I'll wait that all of the jobs will pass and if so I'll update it and will override the jobs